src/utils/converter.ts

```diff
@@ -25,8 +25,12 @@ const categoryNameToId = (categoryName: string) => {
   return null;
 };
 
+const formatGeneration = (generation: string) => {
+  return `${generation.replace('.0', '')}`;
```

Comment:
여기 템플릿 리터럴으로 안해줘도 될 것 같아요!

```suggestion
  return generation.replace('.0', '');
```